README.rst

```diff
@@ -24,7 +24,7 @@ About
 
 coala-bears is a Python package containing all the bears that are officially
 supported by coala. It features more than **65 bears** covering **35 languages**.
-You can see all of them `here <https://github.com/coala-analyzer/bear-docs>`_.
+`Read more about coala-bears  <https://github.com/coala-analyzer/bear-docs/>`_.
```

Comment:
``Here's a list of all the languages supported and the bears available for them``